IdentityCore/src/broker_operation/response/MSIDBrokerOperationGetAccountsResponse.m

```diff
@@ -46,12 +48,29 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)er
     
     if (self)
     {
-        if (![json msidAssertType:NSArray.class ofKey:@"accounts" required:YES error:error])
+        if (![json msidAssertType:NSString.class ofKey:@"accounts" required:NO error:error])
         {
             return nil;
         }
-        NSArray *accountsJson = json[@"accounts"];
         
+        NSString *accountsString = json[@"accounts"];
+        
+        if ([NSString msidIsStringNilOrBlank:accountsString])
+        {
+            self.accounts = @[];
+            self.success = YES;
+            return self;
+        }
+        
+        NSData *jsonData = [accountsString dataUsingEncoding:NSUTF8StringEncoding];
+        NSArray *accountsJson = [NSJSONSerialization JSONObjectWithData:jsonData options:0 error:error];
+                  
+        if (!accountsJson)
```

Comment:
Do we want to make sure `accountsJson` is an array? :)

Reply:
Addressed :)